manifests/e/Eliot/UEExplorer/1.3.1/Eliot.UEExplorer.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Eliot.UEExplorer
+PackageVersion: 1.3.1
+PackageLocale: en-US
+Publisher: Eliot
+PackageName: UE Explorer
+License: MIT License

```

Comment:
I found this on the Project's GitHub page https://github.com/UE-Explorer/UE-Explorer/blob/UE-Explorer/LICENSE.md which shows the product is licensed under GPL-3.0. Is this a mistake here?

Reply:
Oops! :)